SevenSlots/SevenSlots/SevenSlots/View/SlotMachineView.xaml.cs

```diff
@@ -0,0 +1,177 @@
+﻿using Android.Content.PM;
+using SevenSlots.Model;
+using SevenSlots.ViewModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using SevenSlots.Helpers;
+
+namespace SevenSlots.View
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class SlotMachineView : ContentPage
+    {
+        public SlotMachineView()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnAppearing()
+        {
+            base.OnAppearing();
+            MessagingCenter.Send(this, "AllowLandscape");
+        }
+
+        //during page close setting back to portrait
+        protected override void OnDisappearing()
+        {
+            base.OnDisappearing();
+            MessagingCenter.Send(this, "PreventLandscape");
+            (BindingContext as SlotMachineViewModel).UpdateWallet();
+
+            //Save the wallet locally as well
+            string userString = JsonSerializer.Serialize((BindingContext as SlotMachineViewModel).User);
+            Session.GeneralSettings = userString;
+        }
+        private string numberToImageSource(int nr)
+        {
+            if(nr < 3)
+            {
+                return "CherrySlot.png";
+            }
+            else if(nr < 6)
+            {
+                return "GrapeSlot.png";
+            }else if(nr < 9)
+            {
+                return "LemonSlot.png";
+            }else if (nr < 12) 
+            {
+                return "OrangeSlot.png";
+            }else if(nr < 14)
+            {
+                return "SevenSlot.png";
+            }
+            return "SpecialSlot.png";
+        }
```

Comment:
10 else ifs are not the best approach :D . I suggest to save the images in an array or some sort of data structure and get their index by division/some algorithm